src/docs/dates.md

```diff
@@ -120,7 +120,15 @@ display a date with a UTC time zone like:
 Mon, 01 Jan 2018 00:00:00 GMT
 ```
 
-You could add your own `toUTCString` [filter in Liquid](/docs/filters/) to perform the same task.
+On Liquid, you can't do this, though you can create your own `toUTCString` [filter in Liquid](/docs/filters/) to perform the same task, or pass a timezone parameter to your filter to point to the one you desire in your template.
```

Comment:
```suggestion
On Liquid, you can't do this, though you can create your own `toUTCString` [filter in Liquid](/docs/languages/liquid/#filters) to perform the same task, or pass a timezone parameter to your filter to point to the one you desire in your template.
```

I think this specific link is closer to the link text.

Reply:
Honestly, I just kept the original one from the team as I didn't touch that part specifically. Mostly edit it so it made sense to the follow up advice.